eZ/Publish/Core/Persistence/Legacy/Tests/Content/MapperTest.php

```diff
@@ -251,6 +321,30 @@ public function testExtractContentFromRowsMultipleVersions()
         );
     }
 
+    /**
+     * @param string[] $fieldTypeIdentifiers
+     */
+    private function getFieldRegistry(
+        array $fieldTypeIdentifiers = [],
+        ?int $expectedConverterCalls = null
+    ): Registry {
+        $converterMock = $this->createMock(Converter::class);
+        $converterMock->expects(
+            $expectedConverterCalls === null
+            ? self::any()
+            : self::exactly($expectedConverterCalls)
```

Comment:
I usually indent those, but that is a personal preference, so feel free to ignore.
```suggestion
                ? self::any()
                : self::exactly($expectedConverterCalls)
```